.babelrc

```diff
@@ -1,11 +1,27 @@
 {
-  "plugins": [
-    "transform-runtime",
-    "transform-async-generator-functions"
-  ],
-  "presets": [
-    ["env", {
-      "loose": true
-    }]
-  ]
+  "env": {
+    "es5": {
+      "plugins": [
+        "add-module-exports",
+        "transform-runtime",
+        "transform-async-generator-functions"
+      ],
+      "presets": [
+        ["env", {
+          "loose": true
+        }]
+      ]
+    },
+    "es6": {
+      "plugins": []
```

Comment:
Actually I think this is wrong. The library shouldn't try to be full native yet because async generators are nonstandard. It needs syntax-async-generators and transform-async-generator-functions here. Then the async generators will just get transpiled down to separate generators and asyncs (vs all the way to es5).

I guess in the past because it was encouraged not to import the whole library from index there was less of an issue with environments which don't support async generator syntax. That suggests to me that an alternative to transpiling might be to split out a separate async index which is not included in the normal index. Actually, I think that's sensible anyway.